server/src/challenge/challenge.dto.ts

```diff
@@ -3,10 +3,21 @@ export interface CompletedChallengeDto {
   challengeId: string;
 }
 
+export enum LocationTypeDto {
+  EngQuad = 'ENG_QUAD',
```

Comment:
Here is where the conflict comes from, make sure to go over this and updateapi again